README.md

```diff
@@ -14,6 +14,11 @@ We worked hard to come up with a project idea during our first sprint and finall
 The component tree was put together using ideas and comments from all team members and is shown below.
 ![Component tree](./userdesign/component_tree.png)
 
+### Wireframes
+Wireframes for frontend design are below
+![Wireframes](./userdesign/wireframe.png)
+
+
 ### Collaboration
 
 The app was designed and worked on by 6 team members.  Tasks were divided into front end and back end.  Valeria was the product owner and held meetings at least 3 times a week as well as doing some backend coding.  Three members of the team, Valeria, Justin and Robi were engaged working the backend with Robi working on deployment.  Four members of the team Jena, Shanis, Debbie and Becky worked on frontend tasks.  Valeria created a Jira board where we could all enter and assign tasks to work on.  People chose to work on whatever tasks they felt most comfortable with and they were also able to work on learning new technologies to extend their skills if they chose.  Debbie needed to learn react and typescript as these were new to her having previously coded in Python using Flask and Django.  Becky set up an agile stand up sheet in google docs where people could add accomplishments on a daily basis; this helped with sprint planning.  Our main meeting was on a Friday afternoon where we talked about what had been accomplished in each sprint and planned for the next one, using the agile methodology to plan and manage our work.
```

Comment:
There are 7 team members altogether - sorry I missed this the first time.

Reply:
I would say that we had Retrospective and New Sprint meetings